lib/openshift/metering_config.rb

```diff
@@ -10,5 +10,9 @@ def hive_type(user: nil, cached: true, quiet: false)
       raw_resource(user: user, cached: cached, quiet: quiet).dig('spec', 'storage', 'hive', 'type')
     end
 
+    def namespace(user: nil, cached: true, quiet: false)
```

Comment:
This method is redundant. All `ProjectResource` classes have the `project` method. To get the name you do `obj.project.name`

Reply:
other than that lgtm

Reply:
@akostadinov ok, I took that method out.  Please merge is there are no more issues.